sdk/databoxedge/Azure.ResourceManager.DataBoxEdge/assets.json

```diff
@@ -0,0 +1,6 @@
+{
+  "AssetsRepo": "Azure/azure-sdk-assets",
+  "AssetsRepoPrefixPath": "net",
+  "TagPrefix": "net/databoxedge/Azure.ResourceManager.DataBoxEdge",
```

Comment:
Can you move some tests from this pr https://github.com/Azure/azure-sdk-for-net/pull/35483?

Reply:
This PR's generated code has huge differences with the main, so its test cases cannot move to the main, there are lots of compile error